ably-ios/ARTDefault.h

```diff
@@ -15,6 +15,11 @@
 + (NSString*)realtimeHost;
 + (int)TLSPort;
 
+/**
+ Default in minutes of requested time to live for the token.
+ */
++ (double)ttl;
```

Comment:
NSTimeInterval should alleviate the confusion, shouldn't it? It makes the unit of time explicit.